src/seedu/addressbook/storage/jaxb/AdaptedAddressBook.java

```diff
@@ -23,6 +23,8 @@
     private List<AdaptedPerson> persons = new ArrayList<>();
     @XmlElement
     private List<AdaptedTag> tags = new ArrayList<>();
+    @XmlElement
```

Comment:
Good to dabble around the addressbook code, exposure to storage component may come in useful in Addressbook 4.